draft-ftbs-rats-msg-wrap.md

```diff
@@ -100,15 +103,16 @@ When using CBOR, the value field is serialized as a CBOR bytes string.
 {: #fig-cddl artwork-align="left"
    title="CDDL definition"}
 
-## CBOR Tags derived from CoAP Content Formats {#cbor-tag}
+## CMW CBOR Tags {#cbor-tag}
 
-If a CoAP Content Format exists for the RATS conceptual message, the
+CBOR Tags used as CMW are derived from CoAP Content Format values.
+If a CoAP Content Format exists for a RATS conceptual message, the
```

Comment:
```suggestion
If a CoAP Content Format exists for a CMW, the
```

Reply:
mmm, it's really the fact that a content format is defined for the top-level "conceptual message" that enables the automatic CBOR tagging.